### ISSUE TITLE
ERROR ~  Conda environment file does not exist:

### ISSUE THREAD

**Issue by cahuparo** [nf-core/pairgenomealign#23]
Hi there, 

I am testing this pipeline today. But I got an error when nextflow try to find the conda "${moduleDir}/environment.yml" the file does not exist. 

I am using `conda` as profile. There is a conda-forge recipe for `jq`. Just wanted to let you know. I created an `environment.yml` in that directory and I think it worked.

```
Nov-18 13:35:26.864 [Actor Thread 185] DEBUG nextflow.processor.TaskProcessor - Handling unexpected condition for
  task: name=NFCORE_PAIRGENOMEALIGN:PAIRGENOMEALIGN:MULTIQC_ASSEMBLYSCAN_PLOT_DATA; work-dir=null
  error [java.lang.IllegalStateException]: java.lang.IllegalArgumentException: Conda environment file does not exist: /nfs5/BPP/Grunwald_Lab/home/paradarc/pcap_ny_genomes/scripts/synteny/pairgenomealign/./workflows/../modules/local/environment.yml
Nov-18 13:35:26.882 [Actor Thread 185] ERROR nextflow.processor.TaskProcessor - Error executing process > 'NFCORE_PAIRGENOMEALIGN:PAIRGENOMEALIGN:MULTIQC_ASSEMBLYSCAN_PLOT_DATA'

Caused by:
  Conda environment file does not exist: /nfs5/BPP/Grunwald_Lab/home/paradarc/pcap_ny_genomes/scripts/synteny/pairgenomealign/./workflows/../modules/local/environment.yml

``` 

Thanks, 

Camilo

**Comment by charles-plessy** [comment]
Thanks for the report! It will be fixed with PR #26 !

**Comment by charles-plessy** [comment]
Closed in release `1.1.1`!
